cmd/query/app/static_handler_test.go

```diff
@@ -117,57 +119,67 @@ func TestNewStaticAssetsHandlerErrors(t *testing.T) {
 
 // This test is potentially intermittent
 func TestHotReloadUIConfigTempFile(t *testing.T) {
```

Comment:
Does reload functionality actually depend on the type of file being loaded? It seems it only complicates the test without any real benefits.

Reply:
Probably no, but to keep consistency I did these changes. Will revert this back